packages/docs/src/styles/documentation.css

```
@@ -366,13 +368,112 @@
+  .docs-sidebar-rail.is-resizing::after {
     opacity: 0.8;
   }
+
+  .docs-sidebar-rail.is-collapsed::after {
+    opacity: 0;
+  }
+
+  .docs-sidebar-handle {
+    --docs-bookmark: #e0a94f;
+    --docs-bookmark-hover: #cf9438;
+    position: absolute;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)